src/burn/engine/apply.cpp

```diff
@@ -1511,12 +1555,10 @@ static DWORD CALLBACK CacheProgressRoutine(
     LPCWSTR wzPackageOrContainerId = pProgress->pContainer ? pProgress->pContainer->sczId : pProgress->pPackage ? pProgress->pPackage->sczId : NULL;
     LPCWSTR wzPayloadId = pProgress->pPayload ? pProgress->pPayload->sczKey : NULL;
     DWORD64 qwCacheProgress = pProgress->qwCacheProgress + TotalBytesTransferred.QuadPart;
-    if (qwCacheProgress > pProgress->qwTotalCacheSize)
```

Comment:
Seems like we should keep this. Otherwise, the overall percentage on the next line will go over 100%, right? The assert is fine but maybe just pull it in here?